content/trial_getting_started/set_trial_manager/other-deployments.md

```diff
@@ -7,36 +7,36 @@ alwaysopen = false
 
 {{%children style="h2" description="true"%}}
 
-This page provides download links to all of Cloudify's deployment packages, both Premium and Community.
+This page provides download links to all of {{< param product_name >}}'s deployment packages, both Premium and Community.
 
-Read about the differences between our [Cloudify versions](https://cloudify.co/download).
+Read about the differences between our [{{< param product_name >}} versions](https://cloudify.co/download).
 
 
 
 
-## Cloudify Premium (Premium download / Hosted service trial)
+## {{< param product_name >}} Premium (Premium download / Hosted service trial)
 
 Current Version: 5.1       [Release Notes](https://cloudify.co/release-notes-5-1-2020/)
```

Comment:
Do we want to change links as well?

Alos, there are other links pointing to our repositories in this page. They maybe also need to be parametrized (if that is only on this page, it is possible to add params per page, I did that in `content/developer/writing_widgets/widgets-components.md`).

Reply:
This is custom work that will need to be applied by the integrating vendor.